pkgs/tools/misc/mc/default.nix

```diff
@@ -57,9 +57,14 @@ stdenv.mkDerivation rec {
       --replace /bin/rm ${coreutils}/bin/rm
   '';
 
-  preFixup = ''
+  postFixup = ''
     # remove unwanted build-dependency references
     sed -i -e "s!PKG_CONFIG_PATH=''${PKG_CONFIG_PATH}!PKG_CONFIG_PATH=$(echo "$PKG_CONFIG_PATH" | sed -e 's/./0/g')!" $out/bin/mc
+
+    # libX11.so is loaded dynamically so autopatch doesn't detect it
+    patchelf \
```

Comment:
This should be guarded to not run on macos. @Artturin can you open a PR or should I?

Reply:
i can